java/WaysToPrintHello.java

```diff
@@ -0,0 +1,14 @@
+//different ways to print hello world
```

Comment:
Please add information file as stated in the template file, on this line
```sh
// Title  : [Title] 
// Author : [Your-Profile-Username](Your-Profile-Link)
```

Reply:
ok i do